src/main/java/pl/allegro/tech/build/axion/release/domain/VersionSorter.java

```diff
@@ -47,10 +57,16 @@ Result pickTaggedCommit(
             for (String tag : tags) {
                 boolean isNextVersion = nextVersionTagPattern.matcher(tag).matches();
                 if (isNextVersion && (ignoreNextVersionTags || ignoreNextVersionOnHead)) {
+                    if (verbose) {
```

Comment:
Instead of using `IFs` we should just do `logger.debug()`
https://docs.gradle.org/current/userguide/logging.html

Reply:
but what `verbose` actually means? Some random google definition:

> using or expressed in more words than are needed.

So I would not be worried about that. Also currently there seems to be only `4` more debug logs:
https://github.com/search?q=repo%3Aallegro%2Faxion-release-plugin%20logger.debug&type=code

Reply:
I'm more afraid of debug logs from the entire gradel tool + everything else we call at the same time. But I haven't checked what the result would look like now with debug turned on, I can check it so we don't guess

Reply:
I did fast test number o lines:
./gradlew release -Prelease.dryRun -d > debug.log
    6877 debug.log
    
 ./gradlew release -Prelease.dryRun > standart.log
      27 standart.log

Reply:
but maybe it's still ok, maybe it's better than complicating the code, and if someone needs additional information, they will somehow dig through these thousands of lines of logs?

Reply:
I did a quick research and checked in a few other plugins if someone uses the verbose flag or rather the login level. From what I see, however, the login level is used, so I will choose this option. It will be much much simpler